db/functions/packed.py

```diff
@@ -70,6 +70,124 @@ def unpack(self):
         ])
 
 
+class ArrayLengthEquals(DBFunctionPacked):
+    id = 'array_length_equals'
+    name = 'Number of elements is'
+    hints = tuple([
+        hints.returns(hints.boolean),
+        hints.parameter_count(2),
+        hints.parameter(0, hints.array),
+        hints.parameter(1, hints.string_like),
+        hints.mathesar_filter,
+    ])
+
+    def unpack(self):
+        param0 = self.parameters[0]
+        param1 = self.parameters[1]
+        return base.Equal([
+            base.ArrayLength([param0]),
+            param1
+        ])
+
+
+class ArrayLengthGreaterThan(DBFunctionPacked):
+    id = 'array_length_greater_than'
+    name = 'Number of elements is greater than'
+    hints = tuple([
+        hints.returns(hints.boolean),
+        hints.parameter_count(2),
+        hints.parameter(0, hints.array),
+        hints.parameter(1, hints.string_like),
+        hints.mathesar_filter
+    ])
+
+    def unpack(self):
+        param0 = self.parameters[0]
+        param1 = self.parameters[1]
+        return base.Greater([
+            base.ArrayLength([param0]),
+            param1
+        ])
+
+
+class ArrayLengthLessThan(DBFunctionPacked):
+    id = 'array_length_lesser_than'
+    name = 'Number of elements is lesser than'
+    hints = tuple([
+        hints.returns(hints.boolean),
+        hints.parameter_count(2),
+        hints.parameter(0, hints.array),
+        hints.parameter(1, hints.string_like),
+        hints.mathesar_filter
+    ])
+
+    def unpack(self):
+        param0 = self.parameters[0]
+        param1 = self.parameters[1]
+        return base.Lesser([
+            base.ArrayLength([param0]),
+            param1
+        ])
+
+
+class ArrayLengthGreaterorEqual(DBFunctionPacked):
```

Comment:
I think it would make sense to capitalize the `or`: `GreaterorEqual` -> `GreaterOrEqual`.